tests/utils_tests/test_autoreload.py

```diff
@@ -199,6 +199,22 @@ def test_warnoptions(self):
             [sys.executable, '-Werror', __file__, 'runserver']
         )
 
+    @mock.patch('sys.argv', [__file__, 'runserver'])
+    @mock.patch('sys._xoptions', {"utf8": True, "a": "b"})
+    def test_xoptions_with_value(self):
+        self.assertEqual(
+            autoreload.get_child_arguments(),
+            [sys.executable, '-Xutf8', '-Xa=b', __file__, 'runserver']
```

Comment:
Please add a trailing comma:
```suggestion
            [sys.executable, '-Xutf8', '-Xa=b', __file__, 'runserver'],
```